man/loader.conf.xml

```diff
@@ -403,9 +403,27 @@ sbvarsign --attr "${attr}" --key KEK.key --cert KEK.pem --output db.auth db db.e
       <varlistentry>
         <term>reboot-on-error</term>
 
-        <listitem><para>Takes a boolean argument. Enable or disable (the default) auto reboot in case the selected entry fails to start.</para>
+        <listitem><para>Controls auto reboot in case the selected entry fails to start.
 
-        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+        <variablelist>
+          <varlistentry>
+            <term><option>yes</option></term>
+            <listitem><para>Reboot the system if the selected boot entry failed to start.</para></listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term><option>no</option></term>
+            <listitem><para>Don't reboot - pass control back to EFI firmware.</para></listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term><option>auto</option></term>
+            <listitem><para>Perform the reboot if and only if boot counting is enabled for this entry. (This is the default)</para></listitem>
```

Comment:
maybe be more explicit here, also say that the boot counter cannot have hit zero yet. and also maybe give a very brief explanation why. i.e. add "This is typically a safe option, as it ensures a clean measurement log on each boot attempt, but also does not risk an unbounded reboot loop.

Reply:
please end the 2nd sentence in a full stop